plugins/javadoc/src/main/kotlin/javadoc/JavadocPageCreator.kt

```diff
@@ -189,5 +176,11 @@ open class JavadocPageCreator(
         }
         return contents
     }
+
+    private fun DParameter.brief(sourceSet: DokkaSourceSet? = highestJvmSourceSet): List<ContentNode> =
+        briefFromContentNodes(paramsToContentNodes(sourceSet).dropWhile { it is ContentDRILink })
+
+    private fun ContentNode.asJavadocNode(): JavadocSignatureContentNode =
+        (this as ContentGroup).children.firstOrNull() as JavadocSignatureContentNode
```

Comment:
With `firstOrNull` we should return `JavadocSignatureContentNode?`

Reply:
Imho no because it will result in a lot of clutter in page creator and `JavadocPageCreator` will always be used with javadoc signature provider so the output will stay the same. Also if it isn't we will get an exception that something went really wrong. 

I've changed the exception to be more informative